lib/suspenders/app_builder.rb

```diff
@@ -309,6 +309,92 @@ def setup_stylesheets
                 "app/assets/stylesheets/application.scss"
     end
 
+    def install_bourbon_n_friends
+      inject_into_file(
+        'Gemfile',
+        %{\ngem 'bourbon'\ngem 'neat'},
+        after: Regexp.new("gem 'rails', '#{Suspenders::RAILS_VERSION}'")
+      )
+      bundle_command 'install'
+
+      inject_into_file(
+        'app/assets/stylesheets/application.scss',
+%{
+@import "bourbon";
+@import "neat";
+@import "base/grid-settings";
+@import "base/base";
+},
+        after: /@charset "utf-8";/
+      )
+    end
+
+    def install_bootstrap
+      inject_into_file(
+        'Gemfile',
+        %{\ngem 'bootstrap-sass'},
+        after: /gem 'sass-rails',.*$/
+      )
+      bundle_command 'install'
+
+      inject_into_file(
+        'app/assets/stylesheets/application.scss',
+        %{\n@import "bootstrap-sprockets";\n@import "bootstrap";},
+        after: /@charset "utf-8";/
+      )
+
+      inject_into_file(
+        'app/assets/javascripts/application.js',
+        %{\n\/\/= require bootstrap-sprockets},
+        after: /\/\/= require jquery_ujs/
+      )
+    end
+
+    def install_foundation
+      inject_into_file(
+        'Gemfile',
+        %{\ngem 'foundation-rails'},
+        after: /gem 'sass-rails',.*$/
+      )
+
+      inject_into_file(
+        'app/assets/stylesheets/application.scss',
+        %{\n@import "foundation_and_overrides";},
+        after: /@charset "utf-8";/
+      )
+
+      inject_into_file(
+        'app/assets/javascripts/application.js',
+        %{\n\/\/= require foundation},
+        after: /\/\/= require jquery_ujs/
+      )
+
+      inject_into_file(
+        'app/assets/javascripts/application.js',
+        %{\n$(document).foundation();},
+        after: /\/\/= require_tree ./
+      )
+
+      inject_into_file(
+        'app/views/layouts/application.html.erb',
+        %{<%= javascript_include_tag "vendor/modernizr" %>\n},
+        before: /<title>.*$/
+      )
+
+      bundle_command 'install'
+
+      result = run('bundle show foundation-rails', capture: true ).gsub("\n",'')
+      settings_file = File.join(
+        result,
+        'vendor', 'assets', 'stylesheets', 'foundation', '_settings.scss'
+      )
```

Comment:
ya, protip for using `#run` default is to return `true`/`false` if you want the actual return value of the command you're running you need to pass `capture: true`